src/middleware.ts

```diff
@@ -106,3 +107,84 @@ export const withWebhookSignatureVerified =
 
     return handler(request);
   };
+
+export interface DashboardTokenPayload extends JwtPayload {
+  app: string;
+}
+
+export const withJWTVerified =
+  (getAppId: (request: Request) => Promise<string | undefined>): Middleware =>
+  (handler) =>
+  async (request) => {
+    const { [SALEOR_DOMAIN_HEADER]: saleorDomain, "authorization-bearer": token } = request.headers;
+
+    if (token === undefined) {
+      return Response.BadRequest({
+        success: false,
+        message: "Missing token.",
```

Comment:
Maybe we can be explicit on errors, especially in SDK.
How about `Missing token. Check "authorization-bearer" header`